### DISCUSSION TITLE
CoolMod y petición

### DISCUSSION THREAD

**Discussion by GaMbiT-cT** [elpatronaco/pccomponentes-buy-bot#12] [Ideas]
Buenas, llevo algún tiempo peleando con el bot y con el resto de bots, conseguí una gigabyte 3070 reacondicionada... la cual se perdió por el camino, sigo peleando con pccom a ver si me ofrecen alguna nueva.
La cosa es que he estado peleando con el bot de CoolMod y parece que he conseguido algo, te paso cambios para que los apliques si lo ves correcto y si estoy equivocado ver como podría solucionarlo. 

En el login.js de CoolMod linea 18 cambiar submit por Submit, por esa mayúscula no me dejaba loguear

En el buy.js he borrado en la linea 22 .includes('AGOTADO')

Desde entonces tengo el bot funcionando en la página de coolmod, pero claro, no sé si entra stock si lo llegaría a comprar o por el contrario he "roto" el script, la verdad es que no entiendo mucho del tema.

La petición que te quería hacer era para la próxima página, creo que es la que mejor precio da a dia de hoy despues de pccom, es la de vsgamers.es

Un saludos y espero tu respuesta

**Comment by elpatronaco** [comment]
Hey, sobre cambiar submit por Submit, el botón para logearse tiene 2 atributos q son parecidos pero son diferentes, que son type='submit' y name='Submit', en principio debería funcionar correctamente. Sobre la línea 22 del buy estás en lo cierto xd, tendré q pensar una forma para ver si está agotado, pero sí, creo que borrando esa línea lo has roto porque ahora no entrará en el condicional de continuar con la compra. A ver cuando puedo arreglarlo. Y sobre la propuesta de la tienda, de momento estoy preparando el script de aussar, que me he puesto esta mañana y lo tengo casi acabado, y el siguiente script será de Vsgamers
